policy/modules/system/systemd.te

```diff
@@ -1527,8 +1543,14 @@ seutil_libselinux_linked(systemd_user_session_type)
 allow systemd_user_runtime_dir_t self:capability { fowner chown sys_admin dac_read_search dac_override };
 allow systemd_user_runtime_dir_t self:process setfscreate;
 
+list_dirs_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_dir_t, systemd_user_runtime_dir_t)
+read_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_dir_t, systemd_user_runtime_dir_t)
+
```

Comment:
This is doesnt make sense, and it can be removed

Reply:
Turns out this is a typo. `systemd-user-runtime-dir` wants to list the contents of `/run/user/UID/systemd/transient` which is `systemd_user_runtime_unit_t` and remove these files.

Reply:
In fact `systemd_user_runtime_unit_t` should have the `user_runtime_content_type` attribute which will take care of this.